eZ/Publish/Core/MVC/Symfony/Cache/Http/LocalPurgeClient.php

```diff
@@ -9,46 +9,60 @@
 namespace eZ\Publish\Core\MVC\Symfony\Cache\Http;
 
 use eZ\Publish\Core\MVC\Symfony\Cache\PurgeClientInterface;
+use eZ\Publish\Core\MVC\Symfony\Cache\TagAwarePurgeClientInterface;
 use Symfony\Component\HttpFoundation\Request;
 
 /**
- * LocalPurgeClient emulates an Http PURGE request received by the cache store.
- * Handy for mono-server.
+ * LocalPurgeClient emulates an Http PURGE request to be received by the Proxy Tag cache store.
+ * Handy for single-serve using Symfony Proxy..
  */
-class LocalPurgeClient implements PurgeClientInterface
+class LocalPurgeClient implements PurgeClientInterface, TagAwarePurgeClientInterface
 {
     /**
      * @var \eZ\Publish\Core\MVC\Symfony\Cache\Http\ContentPurger
      */
     protected $cacheStore;
 
-    public function __construct(RequestAwarePurger $cacheStore)
+    public function __construct(ContentPurger $cacheStore)
```

Comment:
@andrerom do you remember why you've changed this ?

We use `ContentPurger::purgeAllContent()`, but _also_ `RequestAwarePurger::purgeByRequest()`. Since it's incorrect in both cases, I don't think that this change alone is sufficient.